### PR TITLE
adding example for streaming audio

### PR DESCRIPTION
**Pull request by yogeshg**
Adding an example for streaming audio. This usecase was missing from Google Cloud Speech API. The documentation was misleading and was updated in [PR #3074 for google-cloud-python](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/3074)

### CODE REVIEW

**Comment by jonparrott** [comment]
@gguuss can you take a look?

File: speech/cloud-client/transcribe_stream.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+# author: Yogesh Garg
```
Comment (daspecster) [inline speech/cloud-client/transcribe_stream.py]: @yogeshg I think to be accepted you would need to add the Google copyright comment as can be seen [here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/3ae5f4691de8d03d87a899fda0b1ebce522371e0/speech/cloud-client/transcribe.py#L3-L15).

Your authorship should be in the commits. I don't see it displayed this way in the other files but @gguuss or @jonparrott would know for sure.

File: speech/cloud-client/transcribe_stream.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Example usage:
+    python transcribe_stream resources/audio.raw
+ 
+sources:
+ * https://googlecloudplatform.github.io/google-cloud-python/stable/speech-usage.html#streaming-recognition
+
+"""
+
+import logging
```
Comment (daspecster) [inline speech/cloud-client/transcribe_stream.py]: I'm not 100% sure what this does but I imagine you would want to follow the style [here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/3ae5f4691de8d03d87a899fda0b1ebce522371e0/speech/cloud-client/transcribe.py#L25-L28) with the `# [START import_libraries]`.

File: speech/cloud-client/transcribe_stream.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+logger = logging.getLogger(__name__)
+from google.cloud import speech
+
+SAMPLE_RATE=16000
+speech_client = speech.Client()
+
+def transcribe_stream(stream):
+    logging.debug('{} is closed: {}'.format(str(stream), stream.closed))
+
+    sample = speech_client.sample( stream=stream,
```
Comment (daspecster) [inline speech/cloud-client/transcribe_stream.py]: Extra space before `stream=stream` here.

```diff
- sample( stream=stream)
+ sample(stream=stream)
```

File: speech/cloud-client/transcribe_stream.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                                    encoding=speech.Encoding.LINEAR16,
+                                    sample_rate=SAMPLE_RATE)
+    results = list(sample.streaming_recognize())
+    print 'results len:{}'
```
Comment (daspecster) [inline speech/cloud-client/transcribe_stream.py]: I think this needs to have parentheses for Python 3 support.

```diff
- print 'results len:{}'
+ print('results len: {})
```

File: speech/cloud-client/transcribe_stream.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    for r in results:
```
Comment (daspecster) [inline speech/cloud-client/transcribe_stream.py]: Generally one letter variable names are discouraged.

Simple enough to fix these... /r/result/ and /a/alternative/

File: speech/cloud-client/transcribe_stream.py [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+def parse_args():
+    import argparse
+    p = argparse.ArgumentParser()
+    p.add_argument('filename', type=str)
+    args = p.parse_args()
+    return args
+
+if __name__ == '__main__':
+    args = parse_args()
+    main( args )
```
Comment (daspecster) [inline speech/cloud-client/transcribe_stream.py]:
```diff
- main( args )
+ main(args)
```

**Comment by yogeshg** [comment]
@jonparrott and @gguuss,
I was working on another file that I do not wish to merge, and I also renamed this file.
Please let me know if the updated version is fine or if I should raise another request with just one relevant commit.

**Pull request closed without merging**